kalibro_client/processor/hotspot_metric_result.py

```diff
@@ -0,0 +1,34 @@
+from kalibro_client.base import entity_name_decorator
+from kalibro_client.processor import MetricResult
+from kalibro_client.processor import Repository
+import kalibro_client.miscellaneous.date_metric_result
+
+@entity_name_decorator
+class HotspotMetricResult(MetricResult):
+    def __init__(self, line_number=None, message=None,  *init_args, **init_kwargs):
```

Comment:
I also think `args` and `kwargs` is better, but lots of constructors since to be using the `init_` prefix, probably from copy-pasting template code from other places. We probably need to change all the usages to make things consistent.